client/src/components/App.jsx

```diff
@@ -1,11 +1,14 @@
-import React, { Component } from 'react';
+import React, { Fragment } from 'react';
+import RatingTally from './RatingTally.jsx';
+import ReviewList from './ReviewList.jsx';
+import StatChart from './StatChart.jsx';
 
-class App extends Component {
```

Comment:
I'm gonna read about fragment too but is that the reason you now don't need to bring { component } from react?